usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -244,8 +229,7 @@ def handle_column(col, bronze_table_name):
             elif col.handling == "literal":
                 # Use col.source directly as the value
                 return f"{col.source} AS {col.dest_name}"
-            elif col.handling == "parse_string_date":
-                # return f"parse_string_date({bronze_table_name}.{col.source}) AS {col.dest_name}"
+            elif col.handling in ("parse_string_date", "truncate_string_date"):
                 # Require a separator for mmddYYYY, but not for YYYYmmdd, or there is no way to tell them apart
                 # from just regexp.
```

Comment:
Can you include a note in this comment about whether a timestamp is allowed and or expected in these regular expressions?

Reply:
Done